docs/src/components/Nav/Nav.js

```diff
@@ -58,27 +60,22 @@ const Nav = ({ mdxTree, posts }) => {
       }),
   };
 
-  // add tags!
   const mdxTreeSection = mdxTree
-    ? mdxTree
-        ?.filter((branch) => branch.size)
-        .map((item) => ({
-          label: formatLabel(item.name),
-          links: item?.children.map((child) => {
-            return {
-              label: child?.meta?.title || child.name,
-              linkComponent: NavLinkNext,
-              linkComponentProps: {
-                href: child.path.replace('../packages/cascara/src', '/docs'),
-                isActive: child.name === router?.query?.mdx?.[1],
-              },
-              post: child?.meta?.status && (
-                <Tag content={child?.meta?.status} />
-              ),
-            };
-          }),
-        }))
-    : [];
+    ?.filter((branch) => branch.size)
+    .map((item) => ({
+      label: formatLabel(item.name),
+      links: item?.children.map((child) => {
+        return {
+          label: child?.meta?.title || child.name,
+          linkComponent: NavLinkNext,
+          linkComponentProps: {
+            href: child.path.replace('../packages/cascara/src', '/docs'),
+            isActive: child.name === router?.query?.mdx?.[1],
+          },
+          post: child?.meta?.status && <Tag content={child?.meta?.status} />,
+        };
+      }),
+    }));
```

Comment:
No need for an empty array fallback. If there was ever an empty array, that means wherever we are trying to consume that array, we should have a check. I think this was probably done during the refactor though and was no longer needed anyway.